src/ansys/tools/visualization_interface/backends/pyvista/pyvista_interface.py

```diff
@@ -68,15 +74,27 @@ def __init__(
         num_points: int = 100,
         enable_widgets: bool = True,
         show_plane: bool = False,
+        use_qt: bool = False,
         **plotter_kwargs,
     ) -> None:
         """Initialize the plotter."""
         # Generate custom scene if ``None`` is provided
         if scene is None:
             if viz_interface.TESTING_MODE:
-                scene = pv.Plotter(off_screen=True, **plotter_kwargs)
+                if use_qt and _HAS_PYVISTAQT:
+                    scene = pyvistaqt.BackgroundPlotter(off_screen=True)
+                else:
+                    if use_qt and not _HAS_PYVISTAQT:
+                        message = "PyVistaQt dependency is not installed. Install it with " + \
+                                  "`pip install ansys-tools-visualization-interface[pyvistaqt]`."
+                        logger.warning(message)
+                    scene = pv.Plotter(off_screen=True, **plotter_kwargs)
```

Comment:
Same here... all this logic should go into the pyvistaqt backend implementation IMO